src/client/cl_main.c

```diff
@@ -596,6 +603,8 @@ CL_InitLocal(void)
 
 	Cmd_AddCommand("currentmap", CL_CurrentMap_f);
 
+	Cmd_AddCommand("currentview", CL_CurrentView_f);
```

Comment:
I think it would be better to enhance the `viewpos' command instead of introducing a new one. `viewpos` already prints the view origin and yaw. Adding the angles to it would be easy.

Reply:
Updated